gapis/memory/pool.go

```diff
@@ -100,6 +111,49 @@ func (m *Pool) String() string {
 	return strings.Join(l, "\n")
 }
 
+// New creates and returns a new Pool and its id.
```

Comment:
`// NewPool...`
I think I actually prefer `New()` though, given that the type is `Pools`.